dev/CONTRIBUTING.md

```diff
@@ -16,30 +16,24 @@ If you can't find an existing issue, [create a new issue](https://github.com/For
 * [clone this bin](http://jsbin.com/qekike/edit)
 * ...[or fork this fiddle](https://jsfiddle.net/5g20p8vd/6/).
 
-### 2. Fix bugs
+## 2. Fix bugs
 
 Feel like writing some code? The best way to get familiar with the code base is to fix a bug!
 
 `Option a)` You can browse bugs repo by repo:
```

Comment:
I might eliminate "options" and lead with HuBoard. If you had to give someone one place to go, it'd be that, right? You can then explain that issues are available in the individual repos, too, if someone is interested in a particular repo.